spec/facades/new_authorization_request/default_facade_spec.rb

```diff
@@ -3,7 +3,7 @@
 
   let(:authorization_definition) { instance_double(AuthorizationDefinition, editors: [editor]) }
 
-  let(:editor) { Editor.all.first }
+  let(:editor) { ServiceProvider.all.first }
```

Comment:
```suggestion
  let(:editor) { ServiceProvider.editors.first }
```

related à mon commentaire plus haut

Reply:
je rouvre car il me semble que c'est passé à la trappe